### ISSUE TITLE
Everything is in English

### ISSUE THREAD

**Issue by johnhoran** [johnhoran/Laine#6]
Obviously I should stop being lazy and support whatever language people have on their system.

**Comment by ghost** [comment]
I know Ukrainian and Russian and would like to contribute :)

**Comment by johnhoran** [comment]
Yeah I'd really appreciate that.  I don't really have any language beyond English, so any help I can get I'd appreciate.  I've done a push with everything needed to create the translate files.   There isn't much text, so for right now there isn't too much to translate.

To make a translation go to the directory po and do 

> msginit -l <lang> -i messages.pot

that will make a .po file with the strings that currently need to be translated. 

Thanks

**Comment by eduardohki** [comment]
Hello!

I love this extension and would like to help translate to Portuguese! I think the Extension Description can be translated too (I refer to Gnome Tweak Tool).

Should I just do a Fork?

**Comment by johnhoran** [comment]
Thanks very much, I've just done a merge from your fork there now.

As far as I know there isn't a way to have the description translated into multiple languages unfortunately.  It is something that I've looked around for before, and even the official gnome extensions seem to only have their descriptions in English.  So if you know of any extensions that have managed it somehow, then I'd really like to take a look and see how they did it.  If it is possible then it is something that I would definitely be interested in implementing.  

Anyway thanks again for the translation.
